armi/reactor/reactors.py

```diff
@@ -2284,18 +2293,15 @@ def processLoading(self, cs, dbLoad: bool = False):
                     if a.hasFlags(nonUniformAssems, exact=True):
```

Comment:
It wont let me comment on  line 2280/2289, but you told me those need to be called after the expansion in a past comment and i think you are right

Reply:
Oh, i see now that its also set below. Is the initial set here needed? if not maybe only set once and set after 
_updateBlockBOLHeights since then  we are doing one less random thing in the _updateBlockBOLHeights

Reply:
Addressed https://github.com/terrapower/armi/pull/823/commits/76cfc4f4095a94773a16c5f0b808838db7f36d4c

Reply:
Addressed https://github.com/terrapower/armi/pull/823/commits/ded838291fa5683863138fec22c72a69339d494d